test/postServiceTest.js

```diff
@@ -0,0 +1,19 @@
+const mocha = require('mocha');
+const sinon = require('sinon');
+const assert = require('assert');
+const Post = require('../models/Post');
+const mongoose = require('../mongoose');
+
+const PostService = require('../services/PostService');
+
+describe('PostService', () => {
+    describe('getOnePost(post_id)', () => {
+        it('should return correct post', () => {
+            const stub = sinon.stub(Post, 'find').returns(5);
+            const sut = new PostService();
+            post_id = sut.getOnePost(5);
+            assert.equal(post_id, 5);
+            stub.restore();
+        });
```

Comment:
There are a couple things you should be testing in this describe section.
1. The function you mocked should be called with the right arguments
2. The return value should be the value returned via the stub